### ISSUE TITLE
Categories

### ISSUE THREAD

**Issue by Chipx0** [server-status-project/server-status#90]
Can we get categories?

**Comment by thnilsen** [comment]
I've done a bit of work on get categories implemented, or service groups as I've called it. The rendering of the services in a hierarchy is still to be resolved as I've not managed to get any good three-view rendering with the 3.4.1 code of bootstrap (would be simpler with v4). 

In the same code I've also made it possible to edit the services as well as adding some detail/help text for each service (#51) that can be displayed via tooltip on the front page. 

The implementation is based on a separate sql table for the service groups, rather than adding a dummy service as master. Anybody got any thoughts on that?

**The Services menu** showing the groups underneath the services (In use by shows the number of services that is grouped on each category/service group)
![server-status-service-overview](https://user-images.githubusercontent.com/10619272/90321823-b4fa4d80-df4d-11ea-8924-8699b0c2f800.png)

**Editing of a service**
![server-status-edit-service](https://user-images.githubusercontent.com/10619272/90321843-e5da8280-df4d-11ea-9318-028b1aa64cc1.png)

**Editing of a group**
![server-status-edit-group](https://user-images.githubusercontent.com/10619272/90321848-f3900800-df4d-11ea-84aa-964ded84b895.png)
